Neos.Neos/Classes/Domain/Model/UserInterfaceMode.php

```diff
@@ -119,6 +119,24 @@ public function setFusionPath($fusionPath)
 
     /**
      * @return string
+     * @deprecated
```

Comment:
Please always add when this was deprecated so we know when to remove it. S.th. like
```@deprecated with version 4.0 – use getFusionPath() instead```